inc/template-functions.php

```diff
@@ -30,3 +30,31 @@ function ampconf_pingback_header() {
 	}
 }
 add_action( 'wp_head', 'ampconf_pingback_header' );
+
+/**
+ * Adds the search for as the last top-level menu item.
+ *
+ * @param string   $items The HTML list content for the menu items.
+ * @param stdClass $args  An object containing wp_nav_menu() arguments.
+ *
+ * @return string $items
+ */
+function ampconf_wp_nav_menu_items( $items, $args ) {
```

Comment:
Could we perhaps have the search form after the nav ul. If it doesn't call to much styling changes, we can then add it to the header template.

Reply:
@ThierryA it definitely takes less styles to hook it into the header menu (not loads less, but less). Since we are not limited on HTML, but are limited on styles, I think this implementation is a little better for this use case.

Reply:
@kopepasah it literally requires to add `float: left;` to the menu, I think we can live with it 😄  (see [this commit](https://github.com/xwp/ampconf/pull/36/commits/efd1f6fc9ab532c49bbd5e6020386c171c175c56)). Is quite beneficial as we can call the search form straight the in `header.php` and avoid having the hook firing for all menus.